com.gitlab.j0chn.nextcloud_password_client.json

```diff
@@ -0,0 +1,137 @@
+{
+    "app-id": "com.gitlab.j0chn.nextcloud_password_client",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "41",
+    "sdk": "org.gnome.Sdk",
+    "command": "nextcloud_password_client",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=all",
+        "--filesystem=xdg-download",
+        "--filesystem=xdg-documents",
+        "--filesystem=xdg-download/nextcloud_password_client",
```

Comment:
this is redundant

Reply:
Also the `xdg-` are part of `home` so all of them are redundant not just this one.